app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php

```diff
@@ -139,7 +139,7 @@ public static function parseList($list, $removeEmpty = true, $ignoreNumerical =
         $choices = [];
 
         if ($list) {
-            foreach ($list as $val => $label) {
+            foreach ((array) $list as $val => $label) {
```

Comment:
I did, but I think it was due to my fields not being correctly.i have tested on new installation at I had no issue. Will remove from this PR